query-planner-js/src/composedSchema/buildComposedSchema.ts

```diff
@@ -186,7 +200,7 @@ directive without an @${ownerDirective.name} directive`,
       if (!fieldDirectiveArgs) continue;
 
       const fieldMetadata: FederationFieldMetadata = {
-        graphName: graphMap[fieldDirectiveArgs?.['graph']]?.name,
+        graphName: graphMap.get(fieldDirectiveArgs.graph)?.name,
```

Comment:
There's no error-checking done here (no differentiating between lack of `graph` and unknown graph). Is that OK?

Thinking about this led me to open https://github.com/apollographql/specs-join/pull/12

Reply:
I think we just wanted the same pattern as above here, so f85a597e

Reply:
Sounds good. I think my spec PR is relevant anyway (if nothing else, there's an inconsistency in the spec right now).